tests/pipelines/audio_diffusion/test_audio_diffusion.py

```diff
@@ -96,6 +96,7 @@ def dummy_vqvae_and_unet(self):
         )
         return vqvae, unet
 
+    @slow
```

Comment:
Is this something we need?

Reply:
Yes this test takes a minute and this model has pretty much 0 usage, so disabling for fast

Reply:
But the decorator is on `test_audio_diffusion`, seemed unrelated :)